cr/cmd/package.go

```diff
@@ -0,0 +1,57 @@
+// Copyright The Helm Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/helm/chart-releaser/pkg/config"
+	"github.com/helm/chart-releaser/pkg/packager"
+	"github.com/spf13/cobra"
+)
+
+// packageCmd represents the package command
+var packageCmd = &cobra.Command{
+	Use:   "package [CHART_PATH] [...]",
+	Short: "Package Helm charts",
+	Long: `This command packages a chart into a versioned chart archive file. If a path
+is given, this will look at that path for a chart (which must contain a
+Chart.yaml file) and then package that directory.
+
+
+If you wish to use advanced packaging options such as creating signed
+packages or updating chart dependencies please use "helm package" instead.`,
```

Comment:
Looks like we could pass any `helm package` options the way you wrote `CreatePackages()`?

Reply:
Anyway not saying we need to add that in this PR. But wondering if we could make it easy to encourage users to sign their packages. Would love to see that happen more.

Reply:
Perhaps, but I didn't have the time to test a bunch of that stuff, so I thought for the first PR we should just focus on the most basic use case.

Reply:
SGTM